packages/manager/apps/telecom/src/app/telecom/pack/slots/voipLine/activation/pack-voipLine-activation.service.js

```diff
@@ -0,0 +1,77 @@
+import drop from 'lodash/drop';
+import forEach from 'lodash/forEach';
+import keys from 'lodash/keys';
+import isString from 'lodash/isString';
+import pick from 'lodash/pick';
+
+/**
+ * Build an array of large x n elements
+ * @param {array} arrayData         Input data (flat array)
+ * @param {Integer} largeParam      Array large
+ * @param {function} callbackParam  Function to format data. if the function return false,
```

Comment:
Same like previous.

Reply:
So after many tries to rewrite this complex code, I decide to remove it (because I didn't need it anymore). Sorry for the last remove, I was lost into this code.